docs/source/top_level_elements/subject.rst

```diff
@@ -0,0 +1,497 @@
+
+.. code-block:: xml
+
+    <subject>
+        <name authority="" valueURI="">
+            <namePart/>
+            </name>
+    </subject>
+
+Topical subjects with @authority values but no URI
+--------------------------------------------------
+All of the Estes Kefauver Crime Documents have controlled LCSH terms within mods:subject/mods:topic, but no URI is given.
+Effort should be made to add in the URIs for these subjects if possible. If not, they will need to be treated as strings.
+This is confined to mods:subject/mods:topic
+
+`Example record - ekcd: <https://digital.lib.utk.edu/collections/islandora/object/ekcd%3A7/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <subject>
+        <topic authority="lcsh">Juvenile delinquency</topic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "Juvenile delinquency" .
+
+
+Topical and name subjects
+-------------------------
+
+Topical subjects without URIs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In migration, subjects with name and topical values will be treated in the same way. Their treatment will be the same
+regardless of whether or not a URI or string is used.
+
+Approximately 12,000 records have at least one subject without a URI. Note that this number is hard to pin down due to
+inconsistencies with where @valueURI is placed on subject nodes. We'll need to treat any of these subjects that aren't able
+to be reconciled as string values. For the postcard collection, the use of dots (Database of the Smokies) as the authority
+makes it impossible to include a URI presently. Other collections with string values that could be
+revisited are: Charlie Daniel Cartoon Collection, Ed Gamble Cartoon Collection, Football Programs, Insurance Company of
+North America Records, the American Civil War Collection, Ramsey Family Papers, Tennessee Documentary History,
+and Volunteer Voices.
+
+Here's an `example record - gamble:123 <https://digital.lib.utk.edu/collections/islandora/object/gamble%3A123/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <topic>Environmentalism</topic>
+    </subject>
+    <subject>
+        <topic>Factory and trade waste--Environmental aspects</topic>
+    </subject>
+    <subject>
+        <topic>Pollution</topic>
+    </subject>
+    <subject>
+        <topic>Knight</topic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "Environmentalism" ;
+        dcterms:subject "Factory and trade waste--Environmental aspects" ;
+        dcterms:subject "Pollution" ;
+        dcterms:subject "Knight" .
+
+Topical subjects with URIs
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note the variations in where @valueURI is placed in the Xpaths listed below.
+
+    mods:subject[@valueURI]/mods:topic
+    mods:subject/mods:topic[@valueURI]
```

Comment:
I don't care that these are not treated as inline code, but I'm pointing it out in case you do.

Reply:
Yes, this way me being lazy. I have just committed this change.